packages/tracing/test/hub.test.ts

```diff
@@ -418,6 +419,7 @@ describe('Hub', () => {
 
           const request = new XMLHttpRequest();
           request.open('GET', '/chase-partners');
+          request.send();
```

Comment:
This errors because we are now actually making the request, to a resource that doesn't exist.

As a quick thing to unblock we can just request `http://example.com` - but we should probably use `SinonFakeServer` like we do to test the xhr transport in https://github.com/getsentry/sentry-javascript/blob/caba96ef557464283b983466e5e490f3545d8c41/packages/browser/test/unit/transports/xhr.test.ts#L18-L24

Reply:
oh somehow the tests did not fail on my machine. They do fail when I disable the Internet. I'm working on this

Reply:
I could not get the tests to work properly with fake servers. But since these tests do not really care about the responses, I have fixed them by waiting for the requests to complete regardless of the result